examples/build.gradle

```diff
@@ -20,6 +20,8 @@ plugins {
 def grpcVersion = "1.46.0"
 def protobufVersion = "3.20.1"
 def protocVersion = protobufVersion
+def junitVersion = "4.11"
```

Comment:
Why create a function here, and not for other versions?

Reply:
you are right